modules/rosa-hcp-firewall-prerequisites.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-sts-aws-prereqs.adoc
+// * rosa_planning/rosa-hcp-prereqs.adoc
+
+ifeval::["{context}" == "rosa-sts-aws-prereqs"]
+:rosa-classic-sts:
+endif::[]
+ifeval::["{context}" == "rosa-hcp-aws-prereqs"]
+:hcp:
+endif::[]
+
+[id="rosa-hcp-firewall-prerequisites_{context}"]
+// Conditionals are to change the title when displayed on the rosa-sts-aws-prereqs page
+ifdef::rosa-classic-sts[]
+= {hcp-title}
+endif::rosa-classic-sts[]
+ifndef::rosa-classic-sts[]
+= AWS firewall prerequisites
+
+This section provides the necessary details that enable you to control egress traffic from your {product-title} with Hosted Control Planes cluster. If you are using a firewall to control egress traffic, you must configure your firewall to grant access to the domain and port combinations below. {product-title} with Hosted Control Planes requires this access to provide a fully managed OpenShift service.
+endif::rosa-classic-sts[]
+
+.Prerequisites
+
+You have configured an Amazon S3 Gateway Endpoint in your AWS Virtual Private Cloud (VPC). This endpoint is required to complete requests from the cluster to the Amazon S3 service. 
+
+.Procedure
+
+. Allowlist the following URLs that are used to install and download packages and tools:
```

Comment:
It just seems strange to me to put the "install" before the download". Don't we need to download something before it is installed. Can we say:
```suggestion
. Allowlist the following URLs that are used to download and install packages and tools:
```